Gopkg.lock

```diff
@@ -146,6 +180,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "cc10c25aee4098042be278519b6a28759804d0a7042283f02c20135076fcc161"
+  inputs-digest = "bb69474f6c7a21658d3a84712f2148b8deedc7046dae134b277e7dfe86f56e3b"
```

Comment:
is that change needed as part of this pr ?

Reply:
Gopkg.lock gets created automatically. I am not sure if I need it.